src/dfe-autocomplete.js

```diff
@@ -17,7 +17,7 @@ const suggestion = (value, options) => {
     const html = option.append ? `<span>${value}</span> ${option.append}` : `<span>${value}</span>`
     return option.hint ? `${html}<br>${option.hint}` : html
   } else {
-    return `<span>No results found</span>`
+    return `<span>No results foundd</span>`
```

Comment:
Typo

Reply:
It was a temporary change, I was testing to see if CI actually fails

Reply:
Awesome work so it is all good